tine20/Tinebase/Auth/Ldap.php

```diff
@@ -62,10 +62,26 @@ public function getLdap()
     public function setIdentity($_identity)
     {
         if ($this->resolveIdentityFromEmailToLogin) {
-            if ($user = Tinebase_User::getInstance()->search(Tinebase_Model_Filter_FilterGroup::getFilterForModel(
-                        Tinebase_Model_FullUser::class, [
-                        ['field' => 'accountEmailAddress', 'operator' => 'equals', 'value' => $_identity]
-                    ]))->getFirstRecord()) {
+            // Throw Exception if filter does not work to change filter below
+            Tinebase_Model_Filter_FilterGroup::$beStrict = true;
+            try {
+                // Maybe Tinebase_User is not assembled finally during ldap auth (issue #7418)
+                $filter = Tinebase_Model_Filter_FilterGroup::getFilterForModel(Tinebase_Model_FullUser::class, [
+                    ['field' => 'accountEmailAddress', 'operator' => 'equals', 'value' => $_identity]
+                ]);
+            }
```

Comment:
i think, accountEmailAddress just isn't defined in the filter Tinebase_Model_FullUserFilter - so it is correct to use the field 'email' of the filter. maybe it will be defined in the future ... but it would remove the code that is not working.

we should just replace accountEmailAddress with 'email' in line 67.

Reply:
Changed accordingly (I will learn to write tests but at a later point in my life, sorry).